dotnet/src/webdriver/Remote/HttpCommandExecutor.cs

```diff
@@ -385,5 +393,43 @@ private class HttpResponseInfo
             public string Body { get; set; }
             public string ContentType { get; set; }
         }
+
+        /// <summary>
+        /// Internal Diagnostic Handler for HttpCommandExecutor Additional Context
+        /// </summary>
+        internal class DiagnosticsHttpHandler : DelegatingHandler
+        {
+            private static readonly ILogger _logger = Log.GetLogger<DiagnosticsHttpHandler>();
+
+            public DiagnosticsHttpHandler(HttpMessageHandler messageHandler)
+                : base(messageHandler)
+            {
+            }
+
+            /// <summary>
+            /// Sends the specified request and returns the associated response.
+            /// </summary>
+            /// <param name="request">The request to be sent.</param>
+            /// <param name="cancellationToken">A CancellationToken object to allow for cancellation of the request.</param>
+            /// <returns>The http response message content.</returns>
+            protected override async Task<HttpResponseMessage> SendAsync(HttpRequestMessage request, CancellationToken cancellationToken)
+            {
+                if (request.Content != null)
+                {
+                    var requestContent = await request.Content.ReadAsStringAsync().ConfigureAwait(false);
+                    _logger.Trace($">> Body: {requestContent}");
+                }
+
+                var response = await base.SendAsync(request, cancellationToken).ConfigureAwait(false);
+
+                if (!response.IsSuccessStatusCode)
+                {
+                    var responseContent = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
```

Comment:
```suggestion
                    var responseContent = await response.Content.ReadAsStringAsync(cancellationToken).ConfigureAwait(false);
```

Reply:
Hi @nvborisenko this seems to be failing in my local when running up the tests. I believe because of the targeted framework `netstandard2.0` 

Failing build (added the cancellationToken as part of the argument)
![image](https://github.com/SeleniumHQ/selenium/assets/36796016/9867efdd-0ce5-41a2-86a6-be7915e9533a)

I don't want to update some of the versions that might affect the overall majority of the application's performance. Do we still want to add the `cancellationToken `as an argument in `ReadAsStringAsync `since `ReadAsStringAsync `doesn't have `overload`?

Passing build without the cancellationToken as an argument
![image](https://github.com/SeleniumHQ/selenium/assets/36796016/2dc2b158-7d54-48d0-bcf6-c2828b9af924)

Reply:
My fault, this overload is not available in `netstartaard2.0`